features/car/approach.feature

```diff
@@ -1,11 +1,11 @@
-@routing @approach_param @testbot
+@routing @approach
 Feature: Approach parameter
 
     Background:
         Given the profile "car"
         And a grid size of 10 meters
 
-    Scenario: Testbot - Start End same approach, option unrestricted for Start and End
+    Scenario: Start End same approach, option unrestricted for Start and End
```

Comment:
Previously the standard has been include the profile name at the beginning of the senario name. Dpn't have a strong opinion about it but might want to be consistent.